hathor/consensus/block_consensus.py

```diff
@@ -171,15 +171,15 @@ def update_voided_info(self, block: Block) -> None:
                     best_score = head_meta.score
                 else:
                     # All heads must have the same score.
-                    assert abs(best_score - head_meta.score) < 1e-10
-            assert isinstance(best_score, (int, float))
+                    assert best_score == head_meta.score
+            assert isinstance(best_score, int)
 
             # Calculate the score.
             # We cannot calculate score before getting the heads.
             score = self.calculate_score(block)
 
             # Finally, check who the winner is.
-            if score <= best_score - self._settings.WEIGHT_TOL:
+            if score < best_score:
```

Comment:
Why was the `=` removed? Same question for other checks below.

Reply:
Please open one thread for each case so we double check I'm doing it right. In this case, we are checking if our score is smaller than the best_score (but > and == cases are handled in another part of the code).

Reply:
Ok!

Reply:
@glevco and @jansegre , confirm that you agree that I did the right thing. I'll wait for your confirmation before resolving this conversation.

Reply:
I agree.

Reply:
Resolving this, then